cpp/tests/io/shp/polygon_shapefile_reader_test.cpp

```diff
@@ -72,14 +75,32 @@ TEST_F(PolygonShapefileReaderTest, ZeroPolygons) { test("empty_poly.shp", {}, {}
 
 TEST_F(PolygonShapefileReaderTest, OnePolygon)
 {
-  test("one_poly.shp", {0}, {0}, {-10, 5, 5, -10, -10}, {-10, -10, 5, 5, -10});
+  test("one_poly.shp", {0}, {0}, {-10, -10, 5, 5, -10}, {-10, 5, 5, -10, -10});
```

Comment:
Instead of changing the test, we should add an extra test case to verify the `reverse=False` case.